src/index.js

```diff
@@ -0,0 +1,81 @@
+import Updates from '../modules/updates';
+import AddRemove from '../modules/add-remove';
+import Lists from '../modules/lists';
+import './style.css';
+
+const addRemove = new AddRemove();
+
+const order = (a, b) => a.index - b.index;
+
+const logList = () => {
+  const dash = document.getElementById('dashboard');
+  dash.innerHTML = '';
+  addRemove.items.sort(order).forEach((item) => {
+    const lists = document.createElement('li');
```

Comment:
In these lines of code,  The "order"  function is used only once, so I think it could be inlined in the sort method call.